xbmc/cores/VideoPlayer/VideoPlayerAudio.cpp

```diff
@@ -53,7 +53,7 @@ CVideoPlayerAudio::CVideoPlayerAudio(CDVDClock* pClock, CDVDMessageQueue& parent
 , m_audioSink(pClock)
 {
   m_pClock = pClock;
-  m_pAudioCodec = NULL;
+  m_pAudioCodec = nullptr;
```

Comment:
No need to call `std::unique_ptr::operator=` here

Reply:
Thanks, updated